themes/classic/templates/checkout/_partials/cart-voucher.tpl

```diff
@@ -3,7 +3,11 @@
     <div class="cart-voucher">
       <ul class="promo-name">
         {foreach from=$cart.vouchers.added item=voucher}
-          <li>{$voucher.name} <a class="pull-xs-right" href="{$voucher.delete_url}" data-link-action="remove-voucher">{l s='Cancel'}</a></li>
+          <li>{$voucher.name}
+            <div class="pull-xs-right">
+              {$voucher.reduction_formated}
+              <a class="pull-xs-right" href="{$voucher.delete_url}" data-link-action="remove-voucher"><i class="material-icons">{l s='delete'}</i></a></li>
```

Comment:
The remove button should be at right of the voucher name. If we let it here, the amounts are not aligned.